cohort-evaluator-spark/Dockerfile

```diff
@@ -53,7 +53,7 @@ ENV SPARK_HOME=${SPARK_HOME}
 # Become root for the Spark installation process
 USER root
 # Perform a full update of all packages to minimize vulnerabilities
-RUN dnf update -y &&\
+RUN dnf update --nobest -y &&\
```

Comment:
Is `--nobest` meant to be here permanently? Or is this just so you could do a test run?

Reply:
This is temporary for now to allow testing, but may become permanent pending outcome of discussions with Frances.